src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2115,5 +2122,18 @@ Optional=Optional
 Required=Required
 Entry\ type\ cannot\ be\ empty.\ Please\ enter\ a\ name.=Entry type cannot be empty. Please enter a name.
 Field\ cannot\ be\ empty.\ Please\ enter\ a\ name.=Field cannot be empty. Please enter a name.
+
 Shared\ database=Shared database
 Lookup=Lookup
+
+Import\ theme=Import theme
+
```

Comment:
Please keep the translations together - just use an empty line for separating groups. Everything to "theme" belongs together here - so no empty lines.